operator/chartdownloader/factory.go

```diff
@@ -4,51 +4,34 @@ import (
 	"context"
 	"fmt"
 	"net/url"
-	"strings"
 
-	"github.com/aws/aws-sdk-go/aws/client"
-	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+	"github.com/pkg/errors"
 	"k8s.io/helm/pkg/proto/hapi/chart"
 )
 
-type ChartDownloader interface {
+type Interface interface {
```

Comment:
I dislike this naming convention, but dislike `chartdownloader.ChartDownloader` even more. I've seen `Interface` being used in helm/k8s libs so 🤷‍♂